akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -348,7 +347,8 @@ private[stream] final class GraphInterpreter(
   val outHandlers: Array[OutHandler], // Lookup table for the outHandler of the connection
   val logics: Array[GraphStageLogic], // Array of stage logics
   val onAsyncInput: (GraphStageLogic, Any, (Any) ⇒ Unit) ⇒ Unit,
-  val fuzzingMode: Boolean) {
+  val fuzzingMode: Boolean,
+  val context: AnyRef) {
```

Comment:
Ah, I thought that shell is the one that hosts multiple interpreters. I guess we will need to sort this out a bit more and document, because currently slightly confusing. Anyway, now you know better than me so proceed as you wish :)